distribution/docs/src/main/resources/templates/build/quickstart.ftl

```diff
@@ -1,6 +1,21 @@
+<#list quickStarts?sort_by("order") as quickStartIntro>
+<#if (quickStartIntro.status == "published" && quickStartIntro.section?contains("quickStart") && quickStartIntro.level?contains("intro"))>
+
+include::${quickStartIntro.file}[]
 <#list quickStarts?sort_by("order") as quickStart>
-<#if (quickStart.status == "published")>
-include::${quickStart.file}[]
 
+<#if (quickStart.status == "published" && quickStart.section?contains("quickStart") && quickStart.level?contains("section") && quickStart.parent == quickStartIntro.title)>
+
+include::${quickStart.file}[leveloffset=+1]
+
+<#list quickStarts?sort_by("order") as subQuickStart>
+<#if (subQuickStart.status == "published" && subQuickStart.section?contains("quickStart") && quickStart.level?contains("subSection") && subQuickStart.parent == subQuickStartIntro.title)>
```

Comment:
we probs don't have a formatter for these, but it could make this look cleaner if we added some newlines to these longer lines. eg
```
<#if (subQuickStart.status == "published" 
    && subQuickStart.section?contains("quickStart") 
    && quickStart.level?contains("subSection") 
    && subQuickStart.parent == subQuickStartIntro.title)>
```